protoc-gen-kotlin/lib/build.gradle.kts

```diff
@@ -54,6 +54,16 @@ tasks {
         kotlinPackage.set("pbandk.gen.pb")
         logLevel.set("debug")
     }
+
+    val generateKotlinTestTypes by registering(KotlinProtocTask::class) {
```

Comment:
What is this task actually compiling? You don't specify an `includeDir` and this PR doesn't include any `.proto` files that would need to be built.

Reply:
It was added so I could add the new test package `protoc-gen-kotlin/lib/src/commonTest/kotlin/CodeGeneratorTest.kt` but if it can be in `pbandk.gen` package then this can be removed

Reply:
Would you have an idea on how to include the newly added test so it runs with `./gradlew test` ?

Reply:
Do you mean running the tests in `CodeGeneratorTest`? Use `./gradlew allTests`, which will correctly find tests in Kotlin Multiplatform projects. The `./gradlew test` task only works for JVM projects (and Kotlin Multiplatform projects configured with `kotlin.jvm.withJava()`, such as pbandk's `:runtime` project).